pkg/agent/metrics/prometheus.go

```diff
@@ -116,6 +116,21 @@ var (
 		[]string{"operation"},
 	)
 
+	// OVSMeterPacketCount is defined as a Gauge and not a Counter,
+	// even though this metric is monotonically increasing (only being reset to 0 on restart).
+	// Because we want to set its value directly using the Set method (using the value provided by OVS),
+	// and using Inc / Add is not convenient.
+	OVSMeterPacketCount = metrics.NewGaugeVec(
+		&metrics.GaugeOpts{
+			Namespace:      metricNamespaceAntrea,
+			Subsystem:      metricSubsystemAgent,
+			Name:           "ovs_meter_packet_count",
+			Help:           "Number of packets dropped by OVS meter. The value is more than 0 when the packets exceed the rate-limit.",
```

Comment:
The metric name sounds like it counts packets hit by the meter, should we name it `ovs_meter_packet_dropped` instead of be clear?

Reply:
Sure, will update.